modules/cli/src/main/java/org/apache/ignite/internal/cli/commands/Options.java

```diff
@@ -266,5 +266,23 @@ public static final class Constants {
         public static final String CLUSTER_CONFIG_FILE_OPTION_SHORT = "-cfgf";
 
         public static final String CLUSTER_CONFIG_FILE_OPTION_DESC = "Path to cluster configuration file";
+
+        public static final String PASSWORD_OPTION = "--password";
+
+        public static final String PASSWORD_OPTION_SHORT = "-p";
+
+        public static final String PASSWORD_OPTION_DESC = "Password to connect to cluster";
+
+        public static final String USERNAME_OPTION = "--username";
+
+        public static final String USERNAME_OPTION_SHORT = "-u";
```

Comment:
We use -u and -p for URL and Profile thighs. Can you please create a ticket for revising all short options and reworking this concept?